packages/@tinacms/mdx/src/parse/acorn.ts

```diff
@@ -78,7 +78,7 @@ const extractAttribute = (
         return extractString(attribute, field)
       }
     case 'object':
-      return extractObject(extractExpression(attribute), field)
+      return extractObject(extractExpression(attribute), field, imageCallback)
```

Comment:
Should we track this (add a todo?) and make an issue to refactor this at a latter stage?